app.js

```diff
@@ -1,31 +1,28 @@
 const express = require('express');
 const path = require('path');
-const txtgen = require('txtgen');
-
+const textsnippet = require('./api/textsnippetRoutes');
 const app = express();
 
 //body parser Midware
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
-
 //PATH to Public folder
 app.use('/static', express.static(path.join(__dirname, 'public')))
 
-// Create a new (GET METHOD) API route /generateText
-// inside this route generate a text snippet using "txtgen" and send it as a response
-app.get('/generateText', (req, res) => {
-    const textsnippet = txtgen.sentence();
-    res.send(textsnippet)
-})
+// posts a textsnippet to the page.
+app.use('/api', textsnippet )
 
+// route to main index page.
 app.get('/', (req, res) => {
     res.sendFile(path.resolve( __dirname, './public/index.html'))
 })
 
+// route to every possible 404.
 app.all('*', (req, res) => {
     res.status(404).send('<h1>Home page not found</h1>')
 })
 
+// PORT for the server.
 const PORT = process.env.PORT || 5000;
 app.listen(PORT, ()=> console.log(`'listening on port ${PORT}....'`));
```

Comment:
Remove the single quotes, because it prints in the console like this:
'listening on port 5000....' (Including the single quotes)